Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/DirectionIndicator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Unity.MLAgentsExamples
+{
+    public class DirectionIndicator : MonoBehaviour
+    {
+        public Transform transformToFollow; //ex: hips or body
+        public Transform targetToLookAt; //target in the scene the indicator will point to
+        public float heightOffset;
+        private Vector3 m_StartingPos;
+        
+        void OnEnable()
+        {
+            m_StartingPos = transform.position;
+        }
+
+        void Update()
+        {
+            transform.position = new Vector3(transformToFollow.position.x, m_StartingPos.y + heightOffset, transformToFollow.position.z);
+            Vector3 m_WalkDir = targetToLookAt.position - transform.position;
+            m_WalkDir.y = 0; //flatten dir on the y
```

Comment:
This code will only work in flat environments where the `y` axis is not used. (Which is all of our current example environments)
Do you think it is work it to generalize this code?

Reply:
yeah i've been thinking about this as well. when we do non-uniform terrain there's more things to consider than flattening the direction here so I was going to wait till we crossed that bridge